docker-compose.yml

```diff
@@ -18,9 +18,9 @@ services:
             # dockerfile: backend.dev.dockerfile
         volumes:
             - .:/htb
-        command: python manage.py runserver 0.0.0.0:5000
+        command: python manage.py runserver 0.0.0.0:7000
         ports:
-            - '5000:5000'
+            - "7000:7000"
         depends_on:
             - db
```

Comment:
would not push any changes to the docker compose if they're just changing the port
if you don't want changes to show up in git status you can specifically set git to ignore a file until you turn it back on without changing gitignore
https://codeclimber.net.nz/archive/2016/12/19/ignoring-files-on-git-but-just-for-a-while-with-assume-unchanged/#:~:text=You%20can%20type%20git%20update,assume%2Dunchanged%20.